index.html

```diff
@@ -85,242 +95,270 @@ <h2>LET'S CONNECT</h2>
-      </form>
-      <div class="bottom_line"></div>
+      <div class="right-desktop">
+        <div class="about">
+          <h1>
+            About <br />
+            Myself
+          </h1>
+        </div>
+        <div class="hello hide-overflow">
+          <p >
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="connect-about">
+          <h2>LET'S CONNECT</h2>
+        </div>
+        <div id="footer-icon" class="footer-icons-about">
+          <ul>
+            <li>
+              <a href="#footer-icon">
+                <img src="./images/github-icon.svg" alt="github logo" />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon">
+                <img src="./images/Linkedin icon.svg" alt="Linkedin logo"
+              />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/hand-peace-icon.svg" alt="peace logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/twitter (2).svg" alt="twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/m-icon.svg" alt="m logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <button  id="btn-get" class="see-pro .btn">Get my resume</button>
       </div>
-    </section>
-</fieldset>
+       <div class="desktopV">
+
+         <div class="frame-lang">
+           <h2 class="language">Languages</h2>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="list-skills-about">
+           <ul>
+             <li>
+              <img class="img-tanne" src="./images/javascript.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/JSdesktopAbout.svg" alt="" />
+             </li>
+             <li>
+              <img class="img-tanne" src="./images/html.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/HTMLdesktopAbout.svg" alt="" />
+            </li>
+             <li>
+              <img class="img-tanne" src="./images/css.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/CSSdesktopAbout.svg" alt="" />
+           </ul>
+         </div>
+         <div class="frame">
+           <p>Frameworks</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="frame">
+           <p class="skill-desktop">Skills</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+       </section>
```

Comment:
- [ ] You have done great work on the languages, skills, and frameworks. 👍 . However, it is not good practice to use the `language texts` as images. This makes the website less accessible to screen readers. I humbly request to only use language icons as images and attach the pictures.
- [ ] In addition, you should ensure that the `img` has `alt` attribute. This is because Alt attributes enable screen readers to read the information about on-page images for the benefit of a person with a complete lack of sight, visually impaired, or who is otherwise unable to view the images on the page.
- [ ] Lastly you should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). I therefore kindly request you to fix the `font` of the titles.

  - Your design 😎 
![bold lng](https://user-images.githubusercontent.com/99394374/215084987-7386129b-c9db-4fae-b6e7-2483b8189fb1.PNG)

  - Figma design 😎 
![bold lng fig](https://user-images.githubusercontent.com/99394374/215085081-f5507a0f-1d77-4f0d-9849-da7318815e36.PNG)